docker-compose/docker-compose.yaml

```diff
@@ -619,31 +621,83 @@ services:
       - 'DATA_SOURCE_NAME=postgres://sg:@codeintel-db:5432/?sslmode=disable'
       - 'PG_EXPORTER_EXTEND_QUERY_PATH=/config/code_intel_queries.yaml'
 
-  # Description: TimescaleDB time-series database for code insights data.
+  # Description: This container will update file permissions on the
+  # `codeinsights` database
+  #
+  # Disk: None
+  # Ports: None
+  #
+  # This container should mount the codeinsights volume, change ownership on the files,
+  # remove any reference to the timescaledb library, and then exit cleanly.
+  # After exiting the `codeinsights-db` service is allowed to start.
+  #
+  codeinsights-chown:
+    container_name: codeinsights-chown
+    image: index.docker.io/sourcegraph/alpine-3.12:insiders@sha256:290c7a045f18b8dab3021e6d5667f751b44c5bd488c408ebd387a802a4bf4876
+    cpus: 0.1
+    mem_limit: '10m'
+    command:
+      - "sh"
+      - "-c"
+      - if [ -d /var/lib/postgresql/data/ ]; then chown -R 999:999 /var/lib/postgresql/data/; sed -r -i "s/[#]*\s*(shared_preload_libraries)\s*=\s*'timescaledb(.*)\'/\1 = '\2'/;s/,'/'/" /var/lib/postgresql/data/pgdata/postgresql.conf; fi
```

Comment:
The sed command fixes https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/777 and is just a reversal of the change run in the timescaledb image (https://github.com/timescale/timescaledb-docker/blob/master/Dockerfile#L79).